tests/chainer_tests/functions_tests/math_tests/test_sum.py

```diff
@@ -2,73 +2,57 @@
 
 import numpy
 
-import chainer
-from chainer.backends import cuda
 from chainer import functions
-from chainer import gradient_check
 from chainer import testing
-from chainer.testing import attr
-from chainer.testing import condition
-import chainerx
 
 
 @testing.parameterize(*testing.product({
     'axis': [None, 0, 1, 2, -1, (0, 1), (1, 0), (0, -1), (-2, 0)],
     'keepdims': [True, False],
     'dtype': [numpy.float16, numpy.float32, numpy.float64],
 }))
-class TestSum(unittest.TestCase):
+@testing.fix_random()
+@testing.inject_backend_tests(
+    None,
+    # CPU tests
+    [
+        {},
+        {'use_ideep': 'always'},
+    ]
+    # GPU tests
+    + testing.product({
+        'use_cuda': [True],
+        'use_cudnn': ['never', 'always'],
```

Comment:
Like the above case, It doesn't seem that cudnn is used in `F.sum`.

Reply:
Thanks for reviewing @crcrpar ! Have made the required changes.